### DISCUSSION TITLE
(Hopefully old) Uncertainty about production capability?

### DISCUSSION THREAD

**Discussion by ChrisSimmons** [JasperFx/marten#3283] [Q&A] [answered]
Hey all!  I noticed [this line here in docs](https://github.com/JasperFx/marten/blame/master/documentation/documentation/schema/exporting.md#L11) (published [here](https://martendb.io/schema/exporting.html)):

> For the moment, Marten is not directly supporting any kind of database migration strategy. To be honest, we're waiting to see how Marten plays out in production before investing in anything like that.

Given the doc in question is from 2016 referencing v0.8 and the product has matured immensely since then, can I assume this is just old verbiage that needs some updating?

**Comment by mysticmind** [comment, marked as answer]
It is old verbiage, will update the text to remove `To be honest, we're waiting to see how Marten plays out in production before investing in anything like that.` and add a reference to https://martendb.io/community/tools-and-libraries.html where we have listed a community built tool from Bloomberg tech which has built a db migration tool named [Sable](https://github.com/bloomberg/sable) based on the Marten schema diff features.
